common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -165,4 +163,8 @@ public static long parseSpaceSize(String spaceSize) {
       throw new IllegalArgumentException("Fail to parse " + ori + " to bytes");
     }
   }
+
+  private FormatUtils() {
```

Comment:
``private FormatUtils() {} // prevent instantiation`` is fewer lines of code and more informative